App/components/texts/InfoTextBox.tsx

```diff
@@ -13,6 +13,7 @@ const offset = 10
 
 const style = StyleSheet.create({
   container: {
+    // flexGrow: 3,
```

Comment:
If we no longer need the style we can remove

Reply:
Fixed.